src/Microsoft.AspNetCore.Server.Kestrel.Core/KestrelServerLimits.cs

```diff
@@ -207,5 +210,26 @@ public TimeSpan RequestHeadersTimeout
                 _maxConcurrentUpgradedConnections = value;
             }
         }
+
+        /// <summary>
+        /// Gets or sets the default timeout period for receiving the request body. Defaults to 2 minutes.
+        /// </summary>
+        public TimeSpan DefaultRequestBodyTimeout
+        {
+            get => _defaultRequestBodyTimeout;
+            set
+            {
+                if (value <= TimeSpan.Zero)
+                {
+                    throw new ArgumentOutOfRangeException(nameof(value), CoreStrings.PositiveTimeSpanRequired);
+                }
+                _defaultRequestBodyTimeout = value;
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets the default request body minimum data rate in bytes/second.
+        /// </summary>
```

Comment:
https://github.com/aspnet/Coherence-Signed/issues/427 calls for a default read timeout. You've replaced that with a data rate, but the rate needs to be enabled by default still.